gluten-core/src/main/java/io/glutenproject/substrait/expression/IntLiteralNode.java

```diff
@@ -28,6 +28,10 @@ public IntLiteralNode(Integer value) {
     this.value = value;
   }
 
+  public Integer getValue() {
```

Comment:
Why it needs to add this method for `IntLiteralNode` and `StringLiteralNode` ? For other xxxLiteralNode ?

Reply:
It is used in RegExpReplaceTransformer. 
- First get value of pos by IntLiteralNode.getValue
- For clickhouse backend, if pos value is 1, then transformation is supported, otherwise throw exception. 
-